### PR TITLE
add tags block, add variables gurdduty finding publishing frequency, add not making lambda mode, add not making ipset mode.

### PR DESCRIPTION
**Pull request by Toshihiro-Tadokoro**
feat: add tags block, add variables gurdduty finding publishing frequency, add not making lambda mode, add not making ipset mode.

### CODE REVIEW

**Review by hirosassa — COMMENTED** [review]
Great changes! Check the minor comments. Thanks.

File: guardduty_slack/aws_s3_bucket.tf [hunk 1]
```diff
@@ -1,5 +1,6 @@
 resource "aws_s3_bucket" "main" {
   bucket_prefix = "${var.s3_bucket_name}-"
+  tags          = var.tags
```
Comment (hirosassa) [inline guardduty_slack/aws_s3_bucket.tf]: [minor suggestion] The code alignments broke consistency in the source code formatting in this repo.
It needs consistency.

Comment (Toshihiro-Tadokoro) [reply guardduty_slack/aws_s3_bucket.tf]: @hirosassa
Thank you for your review comment. I fixed it and confirmed it.

File: guardduty_slack/variables.tf [hunk 2]
```diff
@@ -17,4 +17,18 @@ variable "trusted_ip_cidr_blocks" {
 
 variable "guardduty_slack_webhook_url" {
   description = "Slack webhook URL to post notification. Basically use same URL for all environments because no reason to separate security issue notification."
+  default = ""
```
Comment (saiya) [inline guardduty_slack/variables.tf]: Could you update `description` to describe about empty string (build without slack notification)?

Comment (Toshihiro-Tadokoro) [reply guardduty_slack/variables.tf]: @saiya 
Thank you for your review comment. I updated it `description` and` trusted_ip_cidr_blocks `'s `description` .

**Comment by hirosassa** [comment]
LGTM! Thanks!

**Pull request merged**